tests/test_fullchain.py

```diff
@@ -95,6 +96,7 @@ def make_image():
              psf_poly_params=PSF_POLY_PARAMS_TRUTH, with_stars=False, with_rotation=True, with_noise=False)
 
 
+@unittest.skip('Skipping to avoid libradtran dependency')
```

Comment:
It's a shame you can't skip these based on actual knowledge of libradtran availability rather than switching it off regardless (if you could do that then this patch could be upstream)

Reply:
Agreed. I've filed DM-38259 to address that and upstream these changes.